docs/pages-for-subheaders/azure-marketplace-payg-integration.md

```diff
@@ -0,0 +1,203 @@
+
+:::note
+In our example month, we are using 730 hours; this may differ depending on the number of days in the month and the billing cycle.
+:::
+
+**Static Usage:**
+Using Rancher to manage 10 nodes, for 1 month (730 hours) with no additional nodes added in the month.
+**Bursting Model:**
+Using Rancher to manage 10 nodes for 3 weeks (562 hours) in the month and bursting to 30 nodes for 1 week (168 hours).
+**Transient Cluster:**
+A temporary deployment of Rancher on 20 nodes for 2 weeks (336 hours).
+
+**Are special commercial terms available?**
+
+Depending on the deployment, it may be possible to secure special commercial terms, such as an annual subscription. This will be handled via an Azure private offer, contact SUSE for more information.
+
+**Can my spend on Rancher Prime count towards my MACC Program?**
+
+Yes, it can. Contact your Azure Sales Team for more details.
+
+**How do I purchase Rancher for additional nodes?**
+
+Once Rancher Prime has been deployed from the Azure Marketplace and billing is active, there is no need to make a specific purchase for additional nodes. Billing is dynamic and based on the number of nodes Rancher is managing. Just deploy or on-board additional clusters to Rancher as needed.
+
+**Is this an annual commitment, will it auto-renew?**
+
+By default, the Rancher Prime listing is billed on a monthly cycle, based on usage. Billing is on-going for as long as Rancher Prime is deployed.
+
+It is possible to set up an annual commitment via an Azure Private Offer, these will need to be reviewed and renewed at the end of the term, or the deployment will drop back to the default monthly billing cycle.
+
+### Technical
+
+**Do I need a Kubernetes cluster running in Azure to install Rancher and be billed via the Azure Marketplace?**
+
+Yes. To benefit from monthly billing via Azure, the primary Rancher cluster must be an Azure Kubernetes Service (AKS) cluster running in your Azure account. 
+
+**On which Kubernetes distributions can the Rancher Prime Azure Marketplace listing be deployed?**
+
+The Rancher Prime marketplace listing must be deployed on Azure Kubernetes Service (AKS). Downstream/managed clusters can run any supported Kubernetes platform: RKE, RKE2, EKS, GKE, vanilla Kubernetes, OpenShift, Mirantis Kubernetes Engine, etc. See Supported Platforms for more details.
+
+**What is the deployment mechanism?**
+
+The Rancher Prime marketplace listing is deployed using Azure’s CNAB (with Helm inside).
+
+**What is the easiest way to get started?**
+
+One of the easiest ways to get started is to deploy the Rancher Prime marketplace listing to an existing AKS cluster. Follow the instructions in the usage section of the listing. A Helm chart takes care of installation and billing setup.
+
+**What is the minimum version of Rancher required to support Azure Marketplace billing?**
+
+The minimum version supporting marketplace billing is Rancher 2.7.8.
```

Comment:
2.7.9... it was originally 2.7.8 then someone helpfully pointed out to us that that was not a "Rancher Prime" supported version and we updated it. Sorry for the confusion.